samples/hello++/main.cpp

```diff
@@ -1,25 +1,22 @@
 #include <hal/debug.h>
 #include <hal/video.h>
+#include <string>
+#include <vector>
 #include <windows.h>
 
-class NumberClass{
-private:
-  int hiddenValue = 0;
-public:
-  NumberClass() {hiddenValue = 5;};
-  NumberClass(int v) : hiddenValue(v) {};
-  int getValue() { return hiddenValue; };
-};
-
 int main(void) {
   XVideoSetMode(640, 480, 32, REFRESH_DEFAULT);
 
-  NumberClass defaultFive;
-  NumberClass customSix(6);
+  std::vector<std::string> words;
+  words.emplace_back("Hello");
+  words.emplace_back(" ");
+  words.emplace_back("nxdk!\n");
```

Comment:
This would cause 2 `\n` now.
I'd remove the one from this line.

Reply:
oh, how embarrassing 😨 sorry.